gravitee-apim-portal-webui-next/src/app/applications/application/application-tab-logs/application-tab-logs.component.html

```diff
@@ -1,18 +1,98 @@
 <!--
 
     Copyright (C) 2024 The Gravitee team (http://gravitee.io)
-    
+
     Licensed under the Apache License, Version 2.0 (the "License");
     you may not use this file except in compliance with the License.
     You may obtain a copy of the License at
-    
+
             http://www.apache.org/licenses/LICENSE-2.0
-    
+
     Unless required by applicable law or agreed to in writing, software
     distributed under the License is distributed on an "AS IS" BASIS,
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
 
 -->
-<p>application-tab-logs works!</p>
+@if (logs$ | async; as logs) {
+  @if (logs.length) {
+    <div class="m3-body-medium" i18n="@@applicationLogsExplanation">
+      Logs from all APIs for which this application has an accepted, paused, closed, or resumed subscription. Your administrator may
+      restrict what data you see below.
+    </div>
+    <table mat-table [dataSource]="logs" class="application-logs__table">
+      <ng-container matColumnDef="api">
+        <th mat-header-cell *matHeaderCellDef i18n="@@applicationLogsTableColumnApi" class="m3-title-medium">API</th>
+        <td mat-cell *matCellDef="let log">
+          <div>{{ log.apiName }}</div>
+          <div><span i18n="@@applicationLogsTableVersionV">v</span>{{ log.apiVersion }}</div>
```

Comment:
Why did you add this `v`? An api publisher could have decided to use a `v` in the version.
It would end with : `vV2.6.3`

Reply:
Figma 🤷‍♀️ . I can change it to `Version: ` instead.